spec/system/reports_spec.rb

```diff
@@ -29,41 +29,43 @@
       fill_in 'タイトル', with: '初めての日報'
       fill_in '内容', with: <<~TEXT
         参考にした日報のURLを書きました。
-        http://localhost:3000/reports/#{@alice_report.id}
+        http://localhost:3000/reports/#{@report.id}
       TEXT
 
-      expect do
-        click_on '登録する'
-        assert_text '日報が作成されました。'
-      end.to change { ReportMention.count }.by(1)
+      click_on '登録する'
+      expect(page).to have_content '日報が作成されました。'
 
-      visit report_path(@alice_report)
+      expect(page).to have_selector '.show-item', text: '初めての日報'
+
+      click_on "http://localhost:3000/reports/#{@report.id}"
```

Comment:
```ruby
Capybara.configure do |config|
  config.app_host = 'http://localhost'
  config.server_port = 3000
end
```

↑の設定を入れると、`rails s`で起動した開発サーバーとポートを取り合って、どちらか一方しか実行できなくなりそうです。
なので、この設定は外してください。

その上で、`http://localhost:3000/reports/#{@report.id}`のリンクをクリックするのは諦めて、直接

```suggestion
      visit report_path(@report)
```

のようにして言及先のページを表示するようにしてください。
（もちろんユーザーの視点ではリンクをクリックして遷移するのが自然だが、ここは技術上の制限があるので例外的に諦める）

Reply:
```ruby
Capybara.configure do |config|
  config.app_host = 'http://localhost'
  config.server_port = 3000
end
```

↑の設定を入れると、`rails s`で起動した開発サーバーとポートを取り合って、どちらか一方しか実行できなくなりそうです。
なので、この設定は外してください。